windowmanagers/2bwm/DETAILS

```diff
@@ -0,0 +1,27 @@
+. "$GRIMOIRE"/FUNCTIONS
+           SPELL=2bwm
+if [[ "$TWOBWM_BRANCH" == "scm" ]]; then
+         VERSION=$(date "+%Y%m%d")
+          SOURCE=$SPELL-master.zip
+  FORCE_DOWNLOAD=on
+   SOURCE_URL[0]=https://github.com/venam/${SPELL}/archive/refs/heads/master.zip
```

Comment:
why zip here? that'd require `unzip` in dependencies.

Reply:
Ah I used zip because as far as I know, that's all github has for downloading the master branch (via "Code" > "Download Zip" Is there another path you can use that downloads a copy of the current master branch with a different format?